erpnext/projects/doctype/task/task.py

```diff
@@ -35,6 +35,25 @@ def validate(self):
 		self.validate_status()
 		self.update_depends_on()
 
+	def before_save(self):
+		change_idx = False
+		parent_task = frappe.get_value("Task", filters={"name": self.name}, fieldname=["parent_task"])
```

Comment:
If you want the database value, I think you can do:

```suggestion
		parent_task = self.get_doc_before_save().get("parent_task")
```

But do test this one out

Reply:
@vishdha, test this one out